api-reference/10 UI Widgets/dxScheduler/1 Configuration/recurrenceEditMode.md

```diff
@@ -9,16 +9,30 @@ default: 'dialog'
 Specifies the edit mode for recurring appointments.
 
 ---
-This option accepts the following values.
+This option accepts the following values:
 
-- "dialog"  
- Displays a dialog that suggests to a user to choose between editing the entire series or only the current appointment.
+- *"series"*  
+ Enables a user to edit only entire appointment series.
 
-- "series"  
- Enables an end-user to edit only the entire appointment series.
+- *"occurrence"*  
+ Enables a user to edit only individual appointment instances.
 
-- "occurrence"  
- Enables an end-user to edit only the current appointment.
+- *"dialog"*   
+ Displays a dialog that suggests to a user to choose between editing the entire series or only the individual instance.
+
+The **Scheduler** handles changes made to an instance and a series differently.
```

Comment:
```suggestion
The **Scheduler** handles changes made to an instance and a series as follows:
```

Reply:
@dirk-pieterse, the sentence with the applied edit "The **Scheduler** handles changes made to an instance and a series as follows:" is followed by "If a user edits a recurring appointment instance, two actions are performed on the data objects:". Thus this logic is erroneous because the second sentence should be about an instance and a series, but it's not. Can the orignal sentence be used?

Reply:
"The **Scheduler** handles changes made to an instance and a series differently. If a user edits a recurring appointment instance, two actions are performed on the data objects: ..."
@dirk-pieterse is this variant alright?

Reply:
The variant is valid.